pkg/cluster/ignition/library.go

```diff
@@ -538,12 +539,13 @@ func Proxy(inProxy *types.Proxy, noProxies ...string) (*igntypes.Config, error)
 func OcneUser(sshKey string, sshKeyPath string, password string) (*igntypes.Config, error) {
 	ret := NewIgnition()
 
-	// Only allow either ssh key or path to be specified
-	if sshKey != "" && sshKeyPath != "" {
-		return nil, fmt.Errorf("Cannot specify both an ssh key and a path to an ssh key file")
```

Comment:
I believe this can be an "else if" rather than setting sshKeyPath to the empty string.

Reply:
Sure, I can do it that way